minitext.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
   s.email = "kylerippey@gmail.com"
   s.homepage = "http://github.com/kylerippey/minitext"
 
-  s.add_dependency 'twilio-ruby'
+  s.add_dependency 'twilio-ruby', '> 3.11.0'
 
```

Comment:
Version 3.11.0 works. Since the changes.md for twilio-ruby only goes back to this version, I am hesitant to require a lower version.